### ISSUE TITLE
Follower count

### ISSUE THREAD

**Issue by josephcricket** [justintv/Twitch-API#319]
I've observed that the follower count from the channels (https://api.twitch.tv/kraken/channels/user) object is not actually synched to the real number of followers, and seems to update every 30-60 minutes, whereas the follower count obtained accessing the followers resource (https://api.twitch.tv/kraken/channels/user/follows) is synched to the actual real number of followers at every moment.

Is this a bug? does it have something to do with caching? if so, is there a way to retrieve the real channel info at every moment?

**Comment by hallcp** [comment]
External APIs are cached to reduce load.  They are not intended for polling, and as such, polling them will return the same data until the cached value is replaced.  Can you define what is your application use case for needing rapid updates?

**Comment by josephcricket** [comment]
Of course: my application keeps track of certain channels real time viewer and follower statistics, and I get that information from the streams api which provides me with a Stream object with the viewer count and a Channel object with the follower count. But since the channel follower count is not accurate to the realtime value I don't know if I should do 2 queries (one to streams and another to follows) to get an accurate follower count. Is there a way to get the real time follower count using a single call to the streams api?

Is there a way to get both numbers in a single query?
